specs/Deployment/DeploymentAPI.md

```diff
@@ -179,12 +179,57 @@ they shut down, to refer to the updated framework package.
     }
 ```
 
+#### OnError_*
+
+Additional behavior is available if an error occurs in `DeploymentManager.Initialize()`,
+per the following logic:
+
+```c++
+DeploymentManager::Initialize(...)
+{
+    HRESULT hr = S_OK
+
+    // Only supported in packaged processes
+    if IsPackagedProcess()
```

Comment:
Parens after "if"

Reply:
RECOMMEND: Real code or English logic, to avoid confusion